pyatlan/generator/templates/methods/attribute/auth_policy.jinja2

```diff
@@ -1,7 +1,6 @@
 
         @classmethod
-
-        def __create(cls, name: str) -> {{ entity_def.name }}.Attributes:
+        def create(cls, name: str) -> {{ entity_def.name }}.Attributes:
```

Comment:
I think this was intentionally a private method to prevent anyone trying to use this internal mechanism directly (many things need to be set on a policy just-so in order for it to work as expected, which if someone calls this internal method directly they won't know about)